src/main/resources/CodyBundle.properties

```diff
@@ -44,30 +37,33 @@ chat.rate-limit-error.upgrade=\
     or <a href=\"https://sourcegraph.com/cody/subscription\">learn more</a>.<br><br>\
     (Already upgraded to Pro? Restart your IDE for changes to take effect)\
   </html>
-
 chat.rate-limit-error.explain=\
   <html>\
     <b>Thank you for using Cody so heavily today!</b> \
     To ensure that Cody can stay operational for all Cody users, please come back tomorrow for more chats, commands, and autocompletes. \
     <a href=\"https://sourcegraph.com/docs/cody/core-concepts/cody-gateway#rate-limits-and-quotas\">Learn more.</a>\
   </html>
-
 subscription-tab.chat-rate-limit-error=You've used all your chat messages and commands for the month. Upgrade to Pro for unlimited usage.
 subscription-tab.autocomplete-rate-limit-error=You've used all your autocompletions for the month. Upgrade to Pro for unlimited usage.
 subscription-tab.chat-and-autocomplete-rate-limit-error=You've used all your autocompletions and chats for this month. Upgrade to Cody Pro to get unlimited interactions.
 subscription-tab.cody-pro-label=Cody Pro
 subscription-tab.cody-free-label=Cody Free
 subscription-tab.loading-label=Loading...
-
 UpgradeToCodyProNotification.title.upgrade=You've used up your autocompletes for the month
 UpgradeToCodyProNotification.title.explain=Thank you for using Cody so heavily today!
-
 UpgradeToCodyProNotification.content.upgrade=\
   <html>\
     You''ve used all{0} autocomplete suggestions for the month. \
     Upgrade to Cody Pro for unlimited autocompletes, chats, and commands.<br><br>\
     (Already upgraded to Pro? Restart your IDE for changes to take effect)\
   </html>
 UpgradeToCodyProNotification.content.explain=To ensure that Cody can stay operational for all Cody users, please come back tomorrow for more chats, commands, and autocompletes.
-
 tab.subscription.already-pro=(Already upgraded to Pro? Restart your IDE for changes to take effect)
+context-panel.panel-name="Chat Context"
+context-panel.reindex-button-name=""Trigger Reindexing""
```

Comment:
double quote

Reply:
+ why "Chat Context" is quoted? These quotes are visible in UI

Reply:
This issue concerns other labels (see: "1 local project" is also quoted):

![image](https://github.com/sourcegraph/jetbrains/assets/5013708/ac2148ab-2447-42d4-a01e-bd363423445a)

Reply:
Good point, I will fix it!